engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/details/views/repository-manage-module-streams.html

```diff
@@ -0,0 +1,17 @@
+<span page-title ng-model="repository">{{ 'Manage Module Streams for Repository:' | translate }} {{ repository.name }}</span>
```

Comment:
where is this page? I can't seem to find a way to get to it

Reply:
https://github.com/Katello/katello/pull/7698/files#diff-ffb10a7f4a2f3444955cfad88307f3c1R150

Click on the module stream counts